.github/workflows/ci.yml

```diff
@@ -73,8 +73,9 @@ jobs:
         env:
           LSP: '${{ matrix.lsp }}'
           LSP_EXTRACT_FILE: '${{ matrix.lsp_extract_file }}'
-          # Should match 'name:' up above
-          JOB_NAME: 'Windows (${{ matrix.python }}, ${{ matrix.arch }}${{ matrix.extra_name }})'
```

Comment:
I wonder if this could've been reported to codecov.

Reply:
It could be, the codecov action takes a name parameter. I'm not too sure what it does, so I might add it back in out of caution. Do you know what leaving this out does?

Reply:
I think it is displayed on the UI but maybe what the action autodetects is fine too.

Reply:
I checked and it just showed some arbitrary number. I added back the names (this should help debug mismatching upload counts!)